compiler/src/iree/compiler/Dialect/Stream/Transforms/test/specialize_encodings.mlir

```diff
@@ -182,6 +182,49 @@ module {
 
 // -----
 
+#executable_target_vmvx_bytecode_fb = #hal.executable.target<"vmvx", "vmvx-bytecode-fb", {encoding = #iree_encoding.unspecialized_encoding<123>}>
+#map = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+#device_target_local_0_ = #hal.device.target<"local", {ordinal = 0 : index}, [#executable_target_vmvx_bytecode_fb]> : !hal.device
+#encoding = #iree_encoding.encoding<operand_index = 0 : index, op_type =  matmul, element_types = [f32, f32, f32], user_indexing_maps = [#map, #map1, #map2]>
+module {
```

Comment:
I know this was probably copied from some output, but we need to clean things up before adding to lit tests - for example, the extra `module` and indent level are not required.

Reply:
I thought that the `module` is required because I attached a default affinity attribute to it. Now I realized that we do not need it. https://github.com/iree-org/iree/pull/19964 is the first step of cleanups. I'll do the `iree_encoding.encoding<>` cleanups in a follow-up.